config.yml

```diff
@@ -7,9 +7,15 @@ isAWS: true
 UseSlurm: true
 SafeMode: true
 
+## Kalman filter options
+KalmanMode: true
+UpdateFreqDays: 7
+NudgeFactor: 0.1
+FirstPeriod: 1
```

Comment:
It determines the starting point for the current IMI run, within the user's period of interest. For example, if you want to run 3 monthly inversions for May-June-July 2019 and set the corresponding start/end dates (`20190501` and `20190901`) then choosing `FirstPeriod: 1` will execute all three inversions, starting with May. Setting `FirstPeriod: 2` will produce two inversions, starting with June. The reason for this feature is to facilitate re-running inversions for specific sub-periods of the period of interest (rather than having to run everything).